docs/docs/platform/service-replicas.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: 'Service Replicas'
+sidebar_position: 1
+image: /img/og/platform/service-replicas.png
+---
+
+Service Replicas is a feature that allows you to create multiple replicas of your services, enhancing availability and fault tolerance for your Nhost apps. By distributing user requests among replicas, your apps can handle more traffic and provide a better user experience. This documentation touches on the key aspects of Service Replicas in the context of the Nhost Cloud Platform.
```

Comment:
should we also add the same graph as in the blog post? Maybe a link to the announcement too?

Reply:
done